lambda/index.js

```diff
@@ -8,13 +8,25 @@ const Sharp = require('sharp');
 
 const BUCKET = process.env.BUCKET;
 const URL = process.env.URL;
+const ALLOWED_RESOLUTIONS = process.env.ALLOWED_RESOLUTIONS ? new Set(process.env.ALLOWED_RESOLUTIONS.split(/\s*,\s*/)) : new Set([]);
```

Comment:
How about killing the ternary which could confuse newbies? Maybe something like:

```javascript
const ALLOWED_DIMENSIONS = new Set();

if (process.env.ALLOWED_DIMENSIONS) {
  const dimensions = process.env.ALLOWED_DIMENSIONS.split(/\s?,\s?/);
  dimensions.forEach((dimension) => ALLOWED_DIMENSIONS.add(dimension));
}
```

A little too explicit, maybe.

Reply:
Sure no problem, you can remove the ternary.